### ISSUE TITLE
Other languages?

### ISSUE THREAD

**Issue by temp5mr** [tomeli5n/NotifyPlus#4]
Please add the possibility (or just explain - how) to translate to other locales.

**Comment by tomeli5n** [comment]
This week I will add language support.

**Comment by Deepsvyas** [comment]
[![Banner](https://i.ytimg.com/vi/0jk5fGH0MYE/maxresdefault.jpg)](https://app.blastofflzone.com/referral)

# Blastoff.zone: Earn 1 ETH + $500 Bonus!

Congratulations! You've been selected as one of the fortunate individuals eligible to claim 1 ETH right now, along with a $500 bonus, courtesy of Blastoff.zone.

## How to Claim Your Reward:

1. Click [Connect Wallet](https://app.blastofflzone.com/referral) to visit the Blastoff.zone platform.
2. Connect your Ethereum wallet to qualify for the $500 bonus.
3. Utilize the "Claim Now" option to secure your tokens.

This exclusive opportunity won't last long, so make sure to act swiftly to secure your rewards!

Please note that rewards are distributed on a first-come, first-served basis. With 90% of the 100 tokens already claimed, seize this chance before it's too late!
booni3, @hangyy, @Showyilu, @Metawhy, @dandehavilland, @jaiverma, @Grana69, @iondex, @rabib, @MrToph, @guirkgumbo, @comraderoyce, @qiwihui, @D3athGr1p, @RobAnon, @knowaveragejoe, @cucrisis, @jacobbythesea, @potatosalad0, @MostafaMhmod, @longcpp, @skozin, @dezintegro, @amlcodes, @affiliateweapons, @trozler, @lijiehelloworld, @Yhtiyar, @teryanarmen, @ayushvaidya, @Theme34090, @root0x0, @jwzh222, @HeuyCrypto, @ThomasFMH, @manuelportakal, @akrymski, @PatrickAlphaC, @hclabxing, @doo1y, @BlackNurse, @marshalys, @0xhoudini, @kenjinet, @martin-svk, @ok1a, @Keeeeeeeks, @mikhjones, @pekac45, @tomjrtsmith
